### PR TITLE
feat(events/feed): `rss-feed-implementation` for `events`

### PR DESCRIPTION
**Pull request by aniketmaithani**
- `test case added`

### CODE REVIEW

**Comment by aniketmaithani** [comment]
@theskumar @satyaakam @bhanuvrat review me please :)

**Comment by aniketmaithani** [comment]
Addresses #6

**Comment by theskumar** [comment]
Thanks for the PR, these are some the things I think should be updated/integrated.
- generate both rss  and atom feed, you can just subclass after defining one of them.
- add the feed url in the html via meta
- add subscribe to rss feed button in the page.
- add `item_updateddate`
- modify the base url of feed to be `/feed/events-latest.rss` and `feed/events-latest.atom`
- the current implementation returns all the events, limit to some fixed number say 20 or 50 which is a future event, i don't there is a need to return past events.
- look into http://essfeed.org/index.php/Main_Page and try to integrate it.
- in the tests, generate some events and see those are getting populated. You can get check the number of events returned and the events that shouldn't be present the feed, etc.

Let me know if you have any doubts.

**Comment by aniketmaithani** [comment]
@theskumar I will update my PR. Meanwhile you can have a look at some of the changes that I have made :)

File: fossevents/templates/base.html [hunk 1]
```diff
@@ -15,6 +15,7 @@
       <link rel="stylesheet" type="text/css" href="{% static 'css/normalize.css' %}" />
       <link rel="stylesheet" type="text/x-scss" href="{% static 'css/main.scss' %}" />
       {% endcompress %}
+      <a href="/feed/rss/"> <img align="right" src="{% static 'images/subscribe-button.png' %}" height="70px" width="100px"> </a></img>
```
Comment (theskumar) [inline fossevents/templates/base.html:18]: You can't have `<a>` tag inside  `<head>`. Follow the guide from here on how to add rss meta to website: http://www.petefreitag.com/item/384.cfm

**Comment by ultimatecoder** [comment]
@theskumar This feature is much needed. It seems the branch is merged, but the feature isn't deployed. Let me know if any help is a need. Many thanks!

**Pull request closed without merging**